"blank:2.7:nameerror: name 'data' is not defined i have blank main blank script, that imports functions from another blank file, i get this error when it reaches the function int_brief(data):..traceback (most recent call last):.      file ""test.py"", line 28, in &lt;module&gt;.        int_brief(data).    nameerror: name 'data' is not defined...the below function is being called in the main blank file and it is imported from another blank file that has the fucntions. platform is linux and blank 2.7, all the code that runs till the line that has variable data is working fine:..def int_brief(data):..    commandoutput = commandstatus('show ip int brief',expect_string=r'#').    shellout = redirect_shell().    data = pd.read_fwf(stringio(commandoutput),  widths=[23, 16, 4, 7, 22, 8])  .    return commandoutput, shellout, data...i have tried to run the code without the parameter data in the function int_brief(), but still the error is thrown"
